lib/neopixel/blockingneopixel.cpp

```diff
@@ -0,0 +1,183 @@
+#include "mbed.h"
+#include "blockingneopixel.h"
+
+#define H_1_NS 400

```

Comment:
It would be best practice to make these private `const uint32_t` types instead of definitions, since you aren't using them externally and they don't need to be overwritten.